v8env/test/fetch.pipeline.spec.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+
+import pipeline from '@fly/fetch/pipeline'
```

Comment:
You'll need to add a glob here I think https://github.com/superfly/fly/pull/130/files#diff-9fe450fad459952e98a9293481a9c819R110
```
ERROR in ./v8env/test/fetch.pipeline.spec.js
Module not found: Error: Can't resolve '@fly/fetch/pipeline' in '/Users/md/src/superfly/fly/v8env/test'
resolve '@fly/fetch/pipeline' in '/Users/md/src/superfly/fly/v8env/test'
  Parsed request is a module
  using description file: /Users/md/src/superfly/fly/v8env/package.json (relative path: ./test)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/md/src/superfly/fly/v8env/package.json (relative path: ./test)
    resolve as module
      /Users/md/src/superfly/fly/v8env/test/node_modules doesn't exist or is not a directory
      /Users/md/src/superfly/node_modules doesn't exist or is not a directory
      /Users/md/src/node_modules doesn't exist or is not a directory
      /Users/md/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
      looking for modules in /Users/md/src/superfly/fly/v8env/node_modules
        using description file: /Users/md/src/superfly/fly/v8env/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/md/src/superfly/fly/v8env/package.json (relative path: ./node_modules)
          using description file: /Users/md/src/superfly/fly/v8env/package.json (relative path: ./node_modules/@fly/fetch/pipeline)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/v8env/node_modules/@fly/fetch/pipeline doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/v8env/node_modules/@fly/fetch/pipeline.js doesn't exist
            .png
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/v8env/node_modules/@fly/fetch/pipeline.png doesn't exist
            as directory
              /Users/md/src/superfly/fly/v8env/node_modules/@fly/fetch/pipeline doesn't exist
      looking for modules in /Users/md/src/superfly/fly/node_modules
        using description file: /Users/md/src/superfly/fly/package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: /Users/md/src/superfly/fly/package.json (relative path: ./node_modules)
          using description file: /Users/md/src/superfly/fly/package.json (relative path: ./node_modules/@fly/fetch/pipeline)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/node_modules/@fly/fetch/pipeline doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/node_modules/@fly/fetch/pipeline.js doesn't exist
            .png
              Field 'browser' doesn't contain a valid alias configuration
              /Users/md/src/superfly/fly/node_modules/@fly/fetch/pipeline.png doesn't exist
            as directory
              /Users/md/src/superfly/fly/node_modules/@fly/fetch/pipeline doesn't exist
```

Reply:
That shouldn't need a glob, but you will need to run `build:core` to get it to work.

Reply:
Whoops, it's on my end... removed dist & lib and it's fine now 😭